README.md

```diff
@@ -39,6 +39,12 @@ Append the encoded string to the service URL, for example: `https://mermaid.ink/
 
 You could treat it as normal image and embed everywhere you want.
 
+The images are generated with transparent background by default. To force a background color, append the query parameter `?bgColor=<color>` to the URL. 
+`<color>` is interpreted as hexadecimal by default. It is possible to use [named colors](https://htmlcolorcodes.com/color-names/) by prefixing the color name with `!`:
+* `https://mermaid.ink/img/<encoded_string>?bgColor=FF0000` will generate a PNG with a red background;
+* `https://mermaid.ink/img/<encoded_string>?bgColor=!lightgray` will generate an SVG with a light gray background.
```

Comment:
**question**: Is it a typo? why we get a SVG when we are using named colors?

Reply:
Definitely a bad copy-paste, sorry for that.